scripts/check_compliance.py

```diff
@@ -568,8 +578,14 @@ def main():
 
     :return:
     """
+
+    init_logs()
     args = parse_args()
 
+    if args.loglevel:
+        logger.setLevel(level=args.loglevel)
+    # else: os.environ.get('LOG_LEVEL'); see init_logs()
```

Comment:
Maybe some of this stuff could be moved into `into_logs()`, since it has to do with initializing logging. Could have it take `args` as an argument in that case maybe.

Reply:
Submitted, see new PR #19